.swift-format

```diff
@@ -0,0 +1,18 @@
+{
+    "version": 1,
+    "lineLength": 120,
+    "indentation": {
+        "spaces": 2
+    },
+    "lineBreakBeforeEachArgument": true,
+    "indentConditionalCompilationBlocks": false,
+    "prioritizeKeepingFunctionOutputTogether": true,
+    "rules": {
+        "AlwaysUseLowerCamelCase": false,
+        "AmbiguousTrailingClosureOverload": false,
+        "NoBlockComments": false,
+        "OrderedImports": true,
+        "UseLetInEveryBoundCaseVariable": false,
+        "UseSynthesizedInitializer": false
+    }
```

Comment:
Consider: `blankLinesBetweenImports: false` - I like how the original version separated SDK imports vs. other imports.

Reply:
`blankLinesBetweenImports` didn't work, so I had to disable `OrderedImports` instead

Reply:
Oh bummer :( But sounds good re: `OrderedImports`. Maybe things will be fixed in the future...